Crucible.GameAPI/CrucibleIngredient.cs

```diff
@@ -606,6 +648,8 @@ private GameObject CreateStackItem(CrucibleIngredientStackItem crucibleStackItem
             ifs.colliderInner = colliderInner;
             ifs.NextStagePrefabs = crucibleStackItem.GrindChildren.Select(x => this.CreateStackItem(x, depth + 1)).ToArray();
 
+            var sortGroup = stackItem.AddComponent<SortingGroup>();
+
```

Comment:
Custom ingredient StackItems were missing a sort group. This caused exceptions in other mods and had the possibility of causing exceptions in the base game.